python/pyspark/sql/functions.py

```diff
@@ -1475,15 +1487,29 @@ def get_json_object(col, path):
 
 
 @since(1.6)
-def json_tuple(col, fields):
+def json_tuple(col, *fields):
     """Creates a new row for a json column according to the given field names.
 
     :param col: string column in json format
     :param fields: list of fields to extract
 
-    """
-    sc = SparkContext._active_spark_context
-    jc = sc._jvm.functions.json_tuple(_to_java_column(col), fields)
+    >>> data = [("1", '''{"f1": "value1", "f2": "value2", "f3": 3, "f5": 5.23}'''), \
+          ("2", '''{"f1": "value12", "f3": "value3", "f2": 2, "f4": 4.01}'''), \
+          ("3", '''{"f1": "value13", "f4": "value44", "f3": "value33", "f2": 2, "f5": 5.01}'''), \
+          ("4", None), \
+          ("5", '''{"f1": "", "f5": null}'''), \
+          ("6", '''[invalid JSON string]''')]
+    >>> df = sqlContext.createDataFrame(data, ("key", "jstring"))
+    >>> df.select(df.key, json_tuple(df.jstring, 'f1', 'f2', 'f3', 'f4', 'f5')).collect()
```

Comment:
having a case with only two fields would be better. this one is too complicated as an example right now.